spec/features/admin/index/index_spec.rb

```diff
@@ -0,0 +1,50 @@
+require("rails_helper")
+
+RSpec.describe "Admin dashboard" do
+  before(:each) do
+    @customers = create_list(:customer, 5)
+
+    @customer1_invoice_list = create_list(:invoice, 1, customer: @customers[0])
+    @customer2_invoice_list = create_list(:invoice, 3, customer: @customers[1])
+    @customer3_invoice_list = create_list(:invoice, 5, customer: @customers[2])
+    @customer4_invoice_list = create_list(:invoice, 4, customer: @customers[3])
+    @customer5_invoice_list = create_list(:invoice, 2, customer: @customers[4])
+
+    @customer1_invoice_list.each { |invoice| create(:transaction, invoice:) }
```

Comment:
`create(:transaction, invoice:)` is shorthand for `create(:transaction, invoice: invoice)`